doc/dev/general.md

```diff
@@ -82,7 +82,7 @@ This again uses helm and a released helm chart.
 ```shell
 helm install --name kubecf \
      --namespace kubecf \
-     https://scf-v3.s3.amazonaws.com/scf-3.0.0-82165ef3.tgz \
+     https://kubecf.s3.amazonaws.com/kubecf-v0.0.0-998b961.tgz \
```

Comment:
We should update this after a stable release.

Reply:
I think we should update https://github.com/SUSE/kubecf-docs/ for the user-facing docs, here I guess it's fine as it's just development docs, no?

Reply:
Yes, I will create a PR for kubecf-docs too.